charts/public-service/README.md

```diff
@@ -122,6 +122,18 @@ $ helm upgrade --install \
     . 
 ```
 
+### Set up static IP address
+k8s by default reserves different IP addresses after each restart. In order to have a static one, you need to manually create it.
+This IP address will be later used to create a new load balancer and can be pointed from DNS.
+1. Navigate to `Dashboard -> Networking -> VPC Network -> External IP Addresses -> Reserve a static address`
```

Comment:
Shall we mention GCP here?

Reply:
I added links: https://github.com/commercetools/k8s-charts/pull/16/commits/329c56d3e6f450bc44a61f64ed2aee89b9334a89